src/NuGet.Services.Entities/Package.cs

```diff
@@ -156,6 +156,18 @@ public bool HasReadMe
             }
         }
 
+        /// <summary>
+        /// Signifies whether or not the embedded ReadMe exists
+        /// </summary>
+        [NotMapped]
+        public bool HasEmbeddedReadMe
```

Comment:
I know our codebase is inconsistent between `ReadMe` and `Readme`, but from my understanding we would like to migrate to the `Readme` casing. If so, should we use the `Readme` casing on all new properties/methods you introduce?

```suggestion
        public bool HasEmbeddedReadme
```